controller_manager/src/controller_manager.cpp

```diff
@@ -67,17 +70,7 @@ ControllerManager::ControllerManager(
     RCLCPP_WARN(get_logger(), "'update_rate' parameter not set, using default value.");
   }
 
-  std::string robot_description = "";
```

Comment:
I think fetching the relevant params should stay all in the constructor

Reply:
init_resource_manager can take the robot_description string as an argument for less stateful code